session/handler.go

```diff
@@ -304,7 +304,7 @@ func (h *Handler) adminListIdentitySessions(w http.ResponseWriter, r *http.Reque
 	}
 
 	page, perPage := x.ParsePagination(r)
-	sess, err := h.r.SessionPersister().ListSessionsByIdentity(r.Context(), iID, active, page, perPage, uuid.Nil)
+	sess, err := h.r.SessionPersister().ListSessionsByIdentity(r.Context(), iID, active, page, perPage, uuid.Nil, ExpandEverything)
```

Comment:
Can we add or adjust a test case to verify the devices are correctly returned here?

Reply:
Added devices and assertion to the existing test - https://github.com/ory/kratos/blob/7713be7e2c34fb45acf1748819af6a1fe636b7b8/session/test/persistence.go#L188